docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

```diff
@@ -126,9 +175,14 @@ The mathematical expressions are mainly fall into two categories:
      - :math:`PAF = \frac{E(RR_e)-1}{E(RR_e)}`
      - :math:`E(RR_e) = p \times RR + (1-p)`
  - risk exposure is continuous distributed:
-     - :math:`i = i \times (1-PAF) \times rr^{max(e−tmrel,0)/scalar}`
-     - :math:`PAF = \frac{E(RR_e)-1}{E(RR_e)}`
-     - :math:`E(RR_e) = \int_{lower}^{upper}rr^{max(e−tmrel,0)/scalar}p(e)de`
+     - risk effect has a log-linear "dose-response" relationship with exposure:
+         - :math:`i = i \times (1-PAF) \times rr^{max(e-tmrel,0)/scalar}`
+         - :math:`PAF = \frac{E(RR_e)-1}{E(RR_e)}`
+         - :math:`E(RR_e) = \int_{lower}^{upper}rr^{max(e-tmrel,0)/scalar}p(e)de`
+     - risk effect has a non-log-linear relationship with exposure:
+         - :math:`i = i \times (1-PAF) \times f_{rr}(e)`
```

Comment:
Same comment here: The variable $i$ is being used for two different things, so there should actually be a second variable defined below.